### PR TITLE
Implement ViewModels using FreshMVVM.

### PR DESCRIPTION
**Pull request by fruhnow**
This implements #7 using the MVVMFresh Library.

### CODE REVIEW

File: src/JellyfinMobile/JellyfinMobile/JellyfinMobile/ViewModels/ServerDetailViewModel.cs [hunk 1]
```diff
@@ -1,14 +1,22 @@
-﻿using Jellyfin.Client.Models.Finder;
+﻿using System;
+using Jellyfin.Client.Models.Finder;
+using PropertyChanged;
 
 namespace JellyfinMobile.ViewModels
 {
+    [AddINotifyPropertyChangedInterface]
```
Comment (hypervtechnics) [inline src/JellyfinMobile/JellyfinMobile/JellyfinMobile/ViewModels/ServerDetailViewModel.cs]:
```suggestion
```

Is it really necessary here as the attribute is already applied at the base class?

File: src/JellyfinMobile/JellyfinMobile/JellyfinMobile/ViewModels/ServersViewModel.cs [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
 using System.Collections.ObjectModel;
 using System.Diagnostics;
 using System.Threading.Tasks;
-
+using System.Windows.Input;
 using Xamarin.Forms;
 
 namespace JellyfinMobile.ViewModels
 {
+    [AddINotifyPropertyChangedInterface]
```
Comment (hypervtechnics) [inline src/JellyfinMobile/JellyfinMobile/JellyfinMobile/ViewModels/ServersViewModel.cs]:
```suggestion
```

Same with the base class already having the attribtue.

File: src/JellyfinMobile/JellyfinMobile/JellyfinMobile/Pages/ServerDetailPage.xaml [hunk 3]
```diff
@@ -4,13 +4,13 @@
...
              mc:Ignorable="d"
-             x:Class="JellyfinMobile.Views.ServerDetailView"
+             x:Class="JellyfinMobile.Pages.ServerDetailPage"
              Title="{Binding Title}">
 
     <StackLayout Spacing="20" Padding="15">
         <Label Text="ID" FontSize="Medium" />
         <Label Text="{Binding Server.Id}" d:Text="Server Id" FontSize="Small"/>
-        <Label Text="Nam:" FontSize="Medium" />
+        <Label Text="Name:" FontSize="Medium" />
```
Comment (hypervtechnics) [inline src/JellyfinMobile/JellyfinMobile/JellyfinMobile/Pages/ServerDetailPage.xaml]:
```suggestion
        <Label Text="Name" FontSize="Medium" />
```

**Comment by fruhnow** [comment]
applied suggestions, seems good! TYVM

**Pull request merged**